pkgs/by-name/et/ethercat/common.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib,
+  fetchFromGitLab,
+  stdenv,
+  gitUpdater,
+  autoreconfHook,
+  pkg-config,
+  withDocs ? false,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ethercat";
+  version = "1.6.1";
+
+  src = fetchFromGitLab {
+    owner = "etherlab.org";
+    repo = "ethercat";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-Ki8WUKUW4O9SNFTVIXrWpL6suCu7j2nLQtpWVsUoiK0=";
+    fetchSubmodules = withDocs;
+    leaveDotGit = withDocs;
+  };
+
+  meta = {
+    description = "IgH EtherCAT Master for Linux";
+    homepage = "https://etherlab.org/ethercat";
+    changelog = "https://gitlab.com/etherlab.org/ethercat/-/blob/1.6-devel/NEWS";
```

Comment:
We should substitute the version number in here

Reply:
Fixed in b2a4c30a695f671775395b2786e66b754e2ceb68